com.unity.sg2/Editor/GraphDeltaRegistry/FunctionDefinitions/REF.cs

```diff
@@ -61,14 +61,14 @@ public static readonly List<(string, object)> UVs = new()
             };
             public static readonly List<(string, object)> Matrices = new()
             {
-                ("M", M),
-                ("V", V),
-                ("P", P),
-                ("VP", VP),
-                ("I_M", I_M),
-                ("I_V", I_V),
-                ("I_P", I_P),
-                ("I_VP", I_VP)
+                ("Model", M),
+                ("Inverse Model", I_M),
+                ("View", V),
+                ("Inverse View", I_V),
+                ("Projection", P),
+                ("Inverse Projection", I_P),
+                ("View Projection", VP),
+                ("Inverse View Projection", I_VP)
```

Comment:
Updated a bunch of the display names in here so that they match SG1 and show up in the same order.